index.ts

```diff
@@ -214,8 +215,9 @@ class AudioRecorderPlayer {
    * @returns {Promise<string>}
    */
   resumePlayer = async (): Promise<string> => {
-    if (!this._isPlaying) {
-      this._isPlaying = true;
+    if (!this._isPlaying) return 'No audio playing';
+    if (this._isPause) {
```

Comment:
isPause isn't a good naming. Could you rename this to `isPaused` or `hasPaused`?

Reply:
ok